backends-common/elasticsearch/src/test/java/org/apache/james/backends/es/ElasticSearchConfigurationTest.java

```diff
@@ -66,6 +66,30 @@ public void getNbReplicaShouldReturnDefaultValueWhenMissing() throws Configurati
             .isEqualTo(ElasticSearchConfiguration.DEFAULT_NB_REPLICA);
     }
 
+    @Test
+    public void getWaitForActiveShardsShouldReturnConfiguredValue() throws ConfigurationException {
+        PropertiesConfiguration configuration = new PropertiesConfiguration();
+        int value = 36;
+        configuration.addProperty("elasticsearch.index.waitForActiveShards", value);
+        configuration.addProperty("elasticsearch.hosts", "127.0.0.1");
+
+        ElasticSearchConfiguration elasticSearchConfiguration = ElasticSearchConfiguration.fromProperties(configuration);
+
+        assertThat(elasticSearchConfiguration.getWaitForActiveShards())
+            .isEqualTo(value);
+    }
+
+    @Test
+    public void getWaitForActiveShardsShouldReturnDefaultValueWhenMissing() throws ConfigurationException {
+        PropertiesConfiguration configuration = new PropertiesConfiguration();
+        configuration.addProperty("elasticsearch.hosts", "127.0.0.1");
+
+        ElasticSearchConfiguration elasticSearchConfiguration = ElasticSearchConfiguration.fromProperties(configuration);
+
+        assertThat(elasticSearchConfiguration.getWaitForActiveShards())
+            .isEqualTo(ElasticSearchConfiguration.DEFAULT_WAIT_FOR_ACTIVE_SHARDS);
```

Comment:
what about duplicating the value here so that it's harder to change by accident?

Reply:
What do you mean?

Reply:
I mean, be using the constant in `isEqualTo` you make it easy to change the default value.
As a developer, I would like a test to break when I change a default value otherwise I would think nothing is checking that value.